apps/studio/src/pages/home/wallet/Wallet.tsx

```diff
@@ -89,76 +99,120 @@ const Wallet: FunctionComponent = () => {
     setTab(nextTab);
   };
 
-  if (!wallet) {
-    return <NoConnectedWallet />;
-  }
-
-  return (
-    <Container maxWidth={ false }>
-      <Box mx={ [null, null, 1.5] }>
-        <Box
-          sx={ {
-            alignItems: "center",
-            display: "flex",
-            justifyContent: "space-between",
-            mb: 5,
-          } }
-        >
-          <Typography fontWeight={ 800 } variant="h3">
-            WALLET
-          </Typography>
-          <DisconnectWalletButton />
+  if (isClientConfigLoading) return;
```

Comment:
I made this a blank return for the wallet page during loading of the feature flag. I was thinking of putting the wallet title and or connect button but the different views of this page, between the feature and current state in Studio, are different enough that seeing copy/elements doesn't make sense right now. Have no issue changing the default.

Reply:
The current skeletons are buried in the lower components, mainly for the table loading, I feel like it would be a bit of a mess to try and pull them out and separate them per feature flag logic. Up for suggestions though, but like I mentioned the two states are quite different so they would each require their own skeletons.